RoboFile.php

```diff
@@ -543,12 +529,94 @@ private function isWindows()
 	/**
 	 * Return the correct path for Windows
 	 *
-	 * param    string  $path  - The linux path
+	 * @param   string  $path  - The linux path
 	 *
 	 * @return string
 	 */
 	private function getWindowsPath($path)
 	{
 		return str_replace('/', DIRECTORY_SEPARATOR, $path);
 	}
+
+	/**
+	 * Detect the correct driver for selenium
+	 *
+	 * @return  string the webdriver string to use with selenium
+	 *
+	 * @since version
+	 */
+	public function getWebdriver()
+	{
+		$suiteConfig        = Symfony\Component\Yaml\Yaml::parse(file_get_contents('tests/codeception/acceptance.suite.yml'));
+		$codeceptMainConfig = \Codeception\Configuration::config();
+		$browser            = $suiteConfig['modules']['config']['JoomlaBrowser']['browser'];
+
+		if ($browser == 'chrome')
+		{
+			$driver['type'] = 'webdriver.chrome.driver';
+		}
+		elseif ($browser == 'firefox')
+		{
+			$driver['type'] = 'webdriver.gecko.driver';
+		}
+		elseif ($browser == 'MicrosoftEdge')
+		{
+			$driver['type'] = 'webdriver.edge.driver';
+
+			// Check if we are using Windows Insider builds
+			if ($suiteConfig['modules']['config']['AcceptanceHelper']['MicrosoftEdgeInsiders'])
+			{
+				$browser = 'MicrosoftEdgeInsiders';
+			}
+		}
+		elseif ($browser == 'internet explorer')
+		{
+			$driver['type'] = 'webdriver.ie.driver';
+		}
+
+		// Check if we have a path for this browser and OS in the codeception settings
+		if (isset($codeceptMainConfig['webdrivers'][$browser][$this->getOs()]))
+		{
+			$driverPath = $codeceptMainConfig['webdrivers'][$browser][$this->getOs()];
+		}
+		else
+		{
+			$this->yell('No driver for your browser. Check your browser in acceptance.suite.yml and the webDrivers in codeception.yml');
+
+			// We can't do anything without a driver, exit
+			exit(1);
+		}
+
+		$driver['path'] = $driverPath;
+
+		return '-D' . implode('=', $driver);
+	}
+
+	/**
+	 * Return the os name
+	 *
+	 * @return string
+	 *
+	 * @since version
+	 */
+	private function getOs()
+	{
+		$os = php_uname('s');
+
+		if (strpos(strtolower($os), 'windows') !== false)
+		{
+			$os = 'windows';
```

Comment:
@compojoom Why don't you do a return 'windows'; here? :) 

You know else kills the cute kittens :P